content/en/dashboards/widgets/geomap.md

```diff
@@ -13,7 +13,11 @@ further_reading:
   text: "Notebooks"
 ---
 
-The geomap widget plots geographic data and visualizes it through shaded regions or points.
+The geomap widget visualizes geographic data with shaded regions or points. It can be used to:
+- View user sessions by country.
+- Drill down to see a list of all sessions in a new tab.
```

Comment:
```suggestion
- Filter to see a list of all sessions in a new tab.
```
I usually replace "drill down" with "filter," but "filter" seems incorrect here because you get to see a full list. Maybe we just remove it? "See a list of all sessions in a new tab."